packages/istanbul-lib-coverage/lib/coverage-map.js

```diff
@@ -58,6 +58,19 @@ CoverageMap.prototype.merge = function (obj) {
     });
 };
 /**
+ * filter the coveragemap based on the callback provided
+ * @param {Function (filename)} callback - Returns true if the should be
```

Comment:
Fixed the typo. And added additional information in the pull request description.